t/porting/customized.dat

```diff
@@ -4,21 +4,15 @@
 AutoLoader cpan/AutoLoader/t/02AutoSplit.t bb90cda13b88599ad45de4b45799d5218afcb6d8
 ExtUtils::Constant cpan/ExtUtils-Constant/lib/ExtUtils/Constant/Base.pm 7560e1018f806db5689dee78728ccb8374aea741
 ExtUtils::Constant cpan/ExtUtils-Constant/t/Constant.t 165e9c7132b003fd192d32a737b0f51f9ba4999e
-Filter::Util::Call pod/perlfilter.pod 545265af2f45741a0e59eecdd0cfc0c9e490c1e8
```

Comment:
This customization is still present in `pod/perlfilter.pod`.

Reply:
This is curious. After changing the two Net-Ping tests I ran make test_harness and got a test failure in t/porting/customized.t. I opened up t/porting/customized.dat and saw the instructions about re-running ./perl -I../lib porting/customized.t --regen, after which make test_porting PASSed.

Now, I admit that I did not look at the changes in customized.dat before pushing the branch and creating the merge request. But the fact that there were lots of changes in customized.dat other than those for Net-Ping implies that the --regen program had not been run in a long time and that make test_porting was failing to recognize that.

Does that logic seem correct? If so, how should we proceed?

Reply:
This may be a side effect of the fact that `pod/perlfilter.pod` originates on CPAN: https://metacpan.org/release/RURBAN/Filter-1.64/source/perlfilter.pod.

Reply:
> This customization is still present in `pod/perlfilter.pod`.

I believe this should be resolved with the merge of https://github.com/Perl/perl5/pull/20237 and subsequent refinements to this p.r.